charts/stash/templates/cleaner.yaml

```diff
@@ -24,7 +24,13 @@ spec:
            kubectl delete validatingwebhookconfigurations admission.stash.appscode.com || true; \
            kubectl delete mutatingwebhookconfiguration admission.stash.appscode.com || true; \
            kubectl delete apiservice v1alpha1.admission.stash.appscode.com || true; \
+           kubectl delete apiservice v1alpha1.appcatalog.appscode.com || true; \
```

Comment:
This is created by Kubernetes as a result of AppBinding CRD registration. Why do we need to delete this?